Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.mm

```diff
@@ -473,11 +475,11 @@ - (void)receiveWatchStreamError:(int)errorCode userInfo:(NSDictionary<NSString *
   return _datastore->ActiveTargets();
 }
 
-- (const std::unordered_map<TargetId, QueryData> &)expectedActiveTargets {
+- (const ActiveTargetMap &)expectedActiveTargets {
   return _expectedActiveTargets;
 }
 
-- (void)setExpectedActiveTargets:(const std::unordered_map<TargetId, QueryData> &)targets {
+- (void)setExpectedActiveTargets:(const ActiveTargetMap &)targets {
```

Comment:
You could pass by value and `std::move(targets)` to potentially avoid an O(n) copy of the target map. (The caller would still have to `std::move` to save the copy).

Reply:
Done.